static/app/components/events/eventReplay/replayClipPreview.spec.tsx

```diff
@@ -211,4 +217,21 @@ describe('ReplayClipPreview', () => {
       screen.queryByTestId('replay-details-breadcrumbs-tab')
     ).not.toBeInTheDocument();
   });
+  it('Render the back and forward buttons when we pass in handleBackClick and handleForwardClick', async () => {
+    const handleBackClick = jest.fn();
+    const handleForwardClick = jest.fn();
+    render(
+      <ReplayClipPreview
+        {...defaultProps}
+        handleBackClick={handleBackClick}
+        handleForwardClick={handleForwardClick}
+      />,
+      {features: ['replay-play-from-replay-tab']}
+    );
+
+    await userEvent.click(screen.getByRole('button', {name: 'Back'}));
```

Comment:
think this test is failing because `Back` and `Next` don't match up with the labels in the files where you defined the buttons -- probably need to rename the `aria-labels`